src/base58.h

```diff
@@ -147,7 +147,7 @@ template<typename K, int Size, CChainParams::Base58Type Type> class CBitcoinExtK
     K GetKey() {
         K ret;
         if (vchData.size() == Size) {
-            //if base58 encouded data not holds a ext key, return a !IsValid() key
+            //if base58 encoded data not holds a ext key, return a !IsValid() key
```

Comment:
Good point!

I'm not familiar with this piece of the code - any suggestions on what the intention was here?

> If the base58 encoded data does not hold an ext key, return an !IsValid() key

Could that be the correct guess? :-)